### ISSUE TITLE
How does this project relate to https://github.com/buckket/twtxt?

### ISSUE THREAD

**Issue by asakura42** [jointwt/twtxt#330]

**Comment by prologic** [comment]
Hello @asakura42 👋

> How does this project relate to https://github.com/buckket/twtxt? #330

It doesn't. The project's codename just shares the same name (_maybe one day we'll change it 🤷‍♂️_)

What is this then? This is a server that you run somewhere of your choosing (_self-hosting_). It is effectively a multi-user twtxt client that has a builtin web interface and api. The basic idea is to bring twtxt to the masses and provide a easy to use user experience that is similar to what users expect of "social media". There are however some pretty hard design goals such as: No advertising, no tracking, no personal identifiable information, no likes, retweets/retoots, etc.

This is this based on twtxt itself (_the backing file formats used are in fact twtxt files_), it is 100% compatible with the rest of the Twtxt community at large. We have created a few extensions that are documented at [dev.twtxt.net](https://dev.twtxt.net) that you can read about. Most notably are the idea of having an `avatar.png` next to your feed and the notion of Twt Subjects and Twt Hashes (_content addressing_) used primarily to form "Conversation" chains and replies.

Basically to an average person, this project's software provides a familiar interface for users that are used to things like Twitter™ for example. But it is designed to be easy to self-host, is also decentralised (_because the web is_) and is "federated".

There is also a Mobile App that utilises the API called Goryon available in both the App Store and Play Store.

Hope this helps answer your question!

**Comment by asakura42** [comment]
Thank you for answer!

>Mobile App

Oh, maybe publish it to F-Droid?

**Comment by asakura42** [comment]
@prologic And another stupid question. I'm askin here to avoid creating a new issue:
Does your project make any _unsolicited connections_? I mean, maybe not your project directly, but any dependency? I don't know well about how Golang works so my question is very unprofessional, yes 😀

Maybe some connections to your pod? Or smth else, dunno

**Comment by prologic** [comment]
> Does your project make any _unsolicited connections_? I mean, maybe not your project directly, but any dependency? I don't know well about how Golang works so my question is very unprofessional, yes 😀

Define "unsolicited connections"?


> Maybe some connections to your pod? Or smth else, dunno

If you've read any of my blogs posts [here](https://twtxt.net/blogs/prologic) or [here](https://prologic.blog) the answer is quite simply "fuck no" 🤣 (_maybe it answers your question above too? 🤔_)

**Comment by prologic** [comment]
Put simply, I'm **really** against tracking, targeted advertising (_just to be specific, but I hate advertising in general_), your data is your own ffs, etc.
